meshtastic/interfaces/ble/interface.py

```
@@ -627,6 +628,85 @@ def _handle_malformed_fromnum(self, reason: str, exc_info: bool = False) -> None
+            return
+        except TypeError as exc:
+            if not _is_unexpected_keyword_error(exc, "error_msg"):
+                fallback()
+                return
+        except Exception:  # noqa: BLE001 - notification callbacks must stay best effort
+            fallback()
+            return
+
+        try:
+            safe_execute(handler_thunk, error_msg)
+            return
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)